src/assets/release-notes-1.27.0-alpha.3.json

```diff
@@ -0,0 +1,835 @@
+    "documentation": [
+      {
+        "description": "KEP",
+        "url": "https://github.com/kubernetes/enhancements/blob/b5b3585/keps/sig-node/2053-downward-api-hugepages/README.md",
+        "type": "KEP"
+      }
+    ],
+    "author": "saschagrunert",
+    "author_url": "https://github.com/saschagrunert",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/115721",
+    "pr_number": 115721,
+    "kinds": ["api-change", "feature"],
+    "sigs": ["node", "apps"],
+    "feature": true,
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "115739": {
+    "commit": "731238fb416a2ae73b39063acdf0c6326555a045",
+    "text": "None",
+    "markdown": "None ([#115739](https://github.com/kubernetes/kubernetes/pull/115739), [@heyste](https://github.com/heyste))",
+    "author": "heyste",
+    "author_url": "https://github.com/heyste",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/115739",
+    "pr_number": 115739,
+    "areas": ["conformance"],
+    "kinds": ["cleanup"],
+    "sigs": ["node", "architecture"],
+    "duplicate": true,
+    "do_not_publish": true
+  },
+  "115758": {
+    "commit": "b8688048f8d3cca513ae36b2eb6e15f457a18df3",
+    "text": "```\n\n#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:\n\n\u003c!--\nThis section can be blank if this pull request does not require a release note.\n\nWhen adding links which point to resources within git repositories, like\nKEPs or supporting documentation, please reference a specific commit and avoid\nlinking directly to the master branch. This ensures that links reference a\nspecific point in time, rather than a document that may change over time.\n\nSee here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files\n\nPlease use the following format for linking documentation:\n- [KEP]: \u003clink\u003e\n- [Usage]: \u003clink\u003e\n- [Other doc]: \u003clink\u003e\n--\u003e",
+    "markdown": "```\n  \n  #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:\n  \n  \u003c!--\n  This section can be blank if this pull request does not require a release note.\n  \n  When adding links which point to resources within git repositories, like\n  KEPs or supporting documentation, please reference a specific commit and avoid\n  linking directly to the master branch. This ensures that links reference a\n  specific point in time, rather than a document that may change over time.\n  \n  See here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files\n  \n  Please use the following format for linking documentation:\n  - [KEP]: \u003clink\u003e\n  - [Usage]: \u003clink\u003e\n  - [Other doc]: \u003clink\u003e\n  --\u003e ([#115758](https://github.com/kubernetes/kubernetes/pull/115758), [@sourcelliu](https://github.com/sourcelliu)) [SIG API Machinery]",
```

Comment:
```suggestion
    "markdown": "NONE ([#115758](https://github.com/kubernetes/kubernetes/pull/115758), [@sourcelliu](https://github.com/sourcelliu)) [SIG API Machinery]",
```

Reply:
Leaving behind a few minor comments for this cleanup. There are two more items that need the same processing on this file. Please take a look